source/numerics.tex

```diff
@@ -381,6 +381,9 @@
   }
 }
 \end{codeblock}
+\indexlibrary{\idxcode{std}!\idxcode{literals}}%
```

Comment:
Hmm, I can understand where you're coming from, trying to group the sub-namespaces together under the `std` index entry, but I don't think that's useful to someone looking up the namespace (I would look for `complex_literals`, not for `std`, when looking for the `complex_literals` namespace), and I don't think a particular entry in the index of library names is the right place for a description of the namespace structure of the library. If we want such a description, it should probably go in a subclause of [organization] instead (but it's not obvious to me who the audience for such a description would be).

This approach creates five index entries:

 * std, literals
 * std, complex_literals
 * complex_literals, std
 * std::literals, complex_literals
 * complex_literals, std::literals

I don't really like any of these, and we don't index `literals` at all this way. I think what we want would be either:

 * literals, complex_literals
 * complex_literals, literals

or

 * literals
 * complex_literals, literals

I would index this as just `\indexlibrarymember{literals}{complex_literals}`; that seems most consistent with what we do for class members.